docusaurus/static/js/particleProfiles.js

```diff
@@ -0,0 +1,200 @@
+        ctx.fill();
+
+        ctx.globalAlpha = 1;
+        // clip image to circle
+        ctx.save();
+        ctx.beginPath();
+        ctx.arc(this.x, this.y, this.r, 0, Math.PI * 2, true);
+        ctx.closePath();
+        ctx.clip();
+        ctx.drawImage(this.image, this.x - this.r, this.y - this.r, this.r * 2, this.r * 2);
+        ctx.restore();
+        ctx.fillStyle = "#000"
+
+
+
+
+        var a = this.x - mouse.x;
+        var b = this.y - mouse.y;
+
+        var distance = Math.sqrt(a * a + b * b);
+        if (distance < (radius * 70)) {
+            this.accX = (this.x - mouse.x) / 100;
+            this.accY = (this.y - mouse.y) / 100;
+            this.vx += this.accX;
+            this.vy += this.accY;
+
+
+        }
+
+    }
+
+    function onMouseMove(e) {
+        mouse.x = e.clientX;
+        mouse.y = e.clientY;
+
+
+    }
+
+    function initScene() {
+        ww = canvas.width = window.innerWidth;
+        wh = canvas.height = window.innerHeight;
+
+        ctx.clearRect(0, 0, canvas.width, canvas.height);
+
+        // Create an image object and load the image file
+        var img = new Image();
+        img.src = "/img/strapi.png";
+
+        // Once the image is loaded, draw it on the canvas
+        img.onload = function () {
+            const aspectRatio = img.width / img.height;
+            const padding = 300
+            const newWidth = ww - padding;
+            const newHeight = newWidth / aspectRatio;
+            // center the image
+            ctx.drawImage(img, padding / 2, (wh - img.height) / 2, newWidth, newHeight);
+
+            // Get the image data and create particles
+            var data = ctx.getImageData(0, 0, ww, wh).data;
+            ctx.clearRect(0, 0, canvas.width, canvas.height);
+            ctx.globalCompositeOperation = "screen";
+
+            particles = [];
+            for (var i = 0; i < ww; i += Math.round(ww / 80)) {
+                for (var j = 0; j < wh; j += Math.round(ww / 80)) {
+                    if (data[((i + j * ww) * 4) + 3] > 80) {
+                        particles.push(new Particle(i, j));
+                    }
+                }
+            }
+
+
+            amount = particles.length;
+        };
+    }
+
+    function onMouseClick() {
+        radius++;
+        if (radius === 5) {
+            radius = 0;
+        }
+    }
+
+    function render(a) {
+        requestAnimationFrame(render);
+        ctx.clearRect(0, 0, canvas.width, canvas.height);
+        for (var i = 0; i < amount; i++) {
+            particles[i].render();
+        }
+    };
+    window.addEventListener("resize", initScene);
+    window.addEventListener("mousemove", onMouseMove);
+    window.addEventListener("click", onMouseClick);
+    initScene();
+    requestAnimationFrame(render);
+
+}
+const konamiCode = ['ArrowUp', 'ArrowUp', 'ArrowDown', 'ArrowDown', 'ArrowLeft', 'ArrowRight', 'ArrowLeft', 'ArrowRight', 'KeyB', 'KeyA'];
+let pressed = [];
```

Comment:
Doesn't need to be `let` as it's an array your not modifying it you are mutating it.